### PR TITLE
Update libgit2 to 2ca088b

### PR DESCRIPTION
**Pull request by ethomson**

### CODE REVIEW

File: LibGit2Sharp/packages.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="LibGit2Sharp.NativeBinaries" version="1.0.165" targetFramework="net40" allowedVersions="[1.0.165]" />
-</packages>
+  <package id="LibGit2Sharp.NativeBinaries" version="1.0.183" targetFramework="net40" />
```
Comment (bording) [inline LibGit2Sharp/packages.config]: You're missing the `allowedVersions` attribute here.

**Comment by ethomson** [comment]
Hmm.  Looks like the Unix platforms aren't loading their native libraries...  they're in the nuget and, from the logs, it looks like they're getting deployed correctly.

@bording any ideas why this might happen?

**Comment by bording** [comment]
Hmm, Travis is currently configured to use the latest version of mono, which is now 5.0. I know it was working fine with 4.8, so maybe try modifying the Travis config to run against 4.8 and see if that's what has changed?

**Comment by ethomson** [comment]
Huh.  Good call!

**Comment by bording** [comment]
So now the question is what changed in 5.0 to break this? I'll play around with it a bit to try and figure out what the problem is.

**Pull request merged**